packages/blade/src/components/Tabs/TabItem.web.tsx

```diff
@@ -24,15 +24,10 @@ const StyledTabButton = styled.button<{
   const isFilled = variant === 'filled';
   const orientation = isVertical ? 'vertical' : 'horizontal';
   const border = isVertical ? 'borderLeft' : 'borderBottom';
-  const selectedState = isSelected ? 'selected' : 'unselected';
+  const selectedState: 'selected' | 'unselected' = isSelected ? 'selected' : 'unselected';
```

Comment:
nit: is it not inferred? 🤔

Reply:
Yeah weird, but it isn't doing inference.